src/librustc_codegen_llvm/builder.rs

```diff
@@ -1315,6 +1315,7 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
     }
 
     pub fn add_incoming_to_phi(&self, phi: ValueRef, val: ValueRef, bb: BasicBlockRef) {
+        self.count_insn("addincoming");
```

Comment:
There's already an `addincoming`

https://github.com/rust-lang/rust/blob/c6807bb1b282e0c5398aa4e659dbc165b6f3c81b/src/librustc_codegen_llvm/builder.rs#L831-L834

I wonder wether the we want to differentiate between these two. Otherwise, r=me.

Reply:
I didn't dig too far into it, but it looked like the `phi` gave you an object you can call `add_to_phi` on to add more items to. Both the `phi` and `add_to_phi` generate 1 `AddIncoming` instruction. Additionally, the `phi` call calls `empty_phi` which generates a `BuildPhi` instruction and increments the `emptyphi` instruction.

Reply:
Ah AFAIK these are all debugging counters anyway, so this should be fine!